doc/source/contributing_code.rst

```diff
@@ -60,6 +60,10 @@ both modules in development mode.
 
 You can do this either with :ref:`conda <dev-with-conda>` or :ref:`pip <dev-with-pip>`.
```

Comment:
Actually if you could add a note about only using **one** of them, I think a few people tried doing both?

Reply:
Yeah, that's why I split the two instructions when building MDA but making is extra clear is great.